### PR TITLE
[10.0] [ADD] hr_expense_operating_unit

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
HR Expense with Operating Units
===============================

This module introduces the following features:

* Adds the Operating Unit (OU) to the Expense.

* Security rules are defined to ensure that users can only see the Expense of that Operating Units in which they are allowed access to.

* Adds Operating Unit (OU) to the account moves while generating accounting entries from the expense.

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review.

**Comment by AaronHForgeFlow** [comment]
@nikul-serpentcs Will you include here the commit history of https://github.com/OCA/operating-unit/pull/30/commits ?

**Comment by nikul-serpentcs** [comment]
sure @aheficent

File: hr_expense_operating_unit/tests/test_hr_expense_operating_unit.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+        self.emp = self._create_hr_employee()
+
+        self.hr_expense1 = self._create_hr_expense(
+            self.ou1, self.emp)
+
+        self.hr_expense2 = self._create_hr_expense(
+            self.b2c, self.emp)
+
+        self._post_journal_entries(self.hr_expense1)
```
Comment (AaronHForgeFlow) [inline hr_expense_operating_unit/tests/test_hr_expense_operating_unit.py:55]: This is ok. However, I miss a test that check the operating units for the journal items created.

**Review by AaronHForgeFlow — COMMENTED** [review]
Please include the commit history here https://github.com/OCA/operating-unit/pull/30

**Comment by AaronHForgeFlow** [comment]
Thank you @nikul-serpentcs I'll approve this when the commit history is fixed

**Comment by nikul-serpentcs** [comment]
@aheficent but still open branch here #30 
if it is merged, I will fix it.

**Comment by nikul-serpentcs** [comment]
@aheficent I somehow managed to preserve the commits :smile: 

becoming GIT Guru :pray:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Thank you for attending the comments

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by serpentcs-dev1 — APPROVED** [review]
Functional and Code Review LGTM :+1:

**Comment by AaronHForgeFlow** [comment]
Don't panic, travis stopped but it is not red :) I restarted the job

**Pull request merged**